DataFormats/L1TMuon/src/EMTFHit.cc

```diff
@@ -73,18 +73,28 @@ namespace l1t {
   }
 
   CSCCorrelatedLCTDigi EMTFHit::CreateCSCCorrelatedLCTDigi() const {
-    return CSCCorrelatedLCTDigi(1,
-                                valid,
-                                quality,
-                                wire,
-                                strip,
-                                pattern,
-                                (bend == 1) ? 1 : 0,
-                                bx + CSCConstants::LCT_CENTRAL_BX,
-                                0,
-                                0,
-                                sync_err,
-                                csc_ID);
+    CSCCorrelatedLCTDigi lct = CSCCorrelatedLCTDigi(1,
+                                                    valid,
+                                                    quality,
+                                                    wire,
+                                                    strip,
+                                                    pattern,
+                                                    (bend == 1) ? 1 : 0,
+                                                    bx + CSCConstants::LCT_CENTRAL_BX,
+                                                    0,
+                                                    0,
+                                                    sync_err,
+                                                    csc_ID);
+    bool quart_bit = strip_quart_bit == 1 ? true : false;
+    bool eighth_bit = strip_eighth_bit == 1 ? true : false;
```

Comment:
This is not important, but only if you need to touch the code anyhow, please consider moving to the following  simpler expressions:
```suggestion
    bool quart_bit = strip_quart_bit == 1;
    bool eighth_bit = strip_eighth_bit == 1;
```

Reply:
Thanks for catching that. It's actually not that important for now since EMTF doesn't use the `isRun3()` for LCTs. However, I think it makes sense to add this in for the future and to be safe.